test-runner-frontend/src/Views/DashboardUtils.test.js

```diff
@@ -0,0 +1,807 @@
+import { testRunnerTestUtils } from '../Assets/testRunnerTestUtils';
```

Comment:
there are some [biiiiiiiiiig](https://www.youtube.com/watch?v=9BCBlz8Jq8U) objects in this test file. is there anyway we can use `testRunnerTestUtils` or programmatically create some data?

Reply:
Done.